prow/config/presubmits.yaml

```diff
@@ -949,6 +949,70 @@ presubmits:
           name: cgroup
         - emptyDir: {}
           name: varlibdocker
+  - name: istio-integration-servicemesh-2.4-mc
+    trigger: (?m)^/test( | .* )integration\-maistra\-servicemesh\-mc,?($|\s.*)
+    rerun_command: /test integration-maistra-servicemesh-mc
+    skip_report: false
+    max_concurrency: 2
+    always_run: true
+    branches:
+    - ^maistra-2.4$
+    decorate: true
+    path_alias: istio.io/istio
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - prow/integ-suite-kind.sh
+        - --topology
+        - MULTICLUSTER
+        - test.integration.servicemesh.kube
+        env:
+        - name: GOFLAGS
+          value: -mod=vendor
```

Comment:
remove this

Reply:
hah, copy&paste design pattern always fail :D